db/migrate/20190314201239_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :comments do |t|
+      t.string :commentable_type, allow_nil: false, allow_blank: false
+      t.integer :commentable_id, allow_nil: false, allow_blank: false
+      t.integer :user_id, allow_nil: false, allow_blank: false
+      t.text :body, allow_nil: false, allow_blank: false
+
+      t.timestamps
+    end
```

Comment:
add index for faster lookup:

```
add_index :comments, :user_id
add_index :comments, [:commentable_id, :commentable_type]
```

Reply:
https://mauricio.github.io/2008/09/27/handling-database-indexes-for-rails-polymorphic-associations.html